File: Day6/PalindromeNumber.java
package Day6;
import java.util.Scanner;
public class PalindromeNumber {
    public static boolean getCheckPalindromeNumber(int num){
        int temp = num , reverse = 0;
        while(num != 0){
        reverse = (reverse*10) + (num%10);
        num /= 10;
        }
        return reverse == temp;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number : ");
        int num = sc.nextInt();
        sc.close();
        boolean res = getCheckPalindromeNumber(num);
        if(res)
            System.out.println(num + " is a Palindrome number.");
        else
            System.out.println(num + " is not a Palindrome number.");
    }
}

File: Day6/PalindromeNumber.cpp
#include<iostream>
using namespace std;

bool getCheckPalindromeNumber(int num){
    int temp = num;
    int reverse = 0;
    while(num != 0){
        reverse = (reverse*10) + (num%10);
        num = num /10;
    }
    return reverse == temp;
}
int main(){
    int num;
    cout<<"\nEnter the number: ";
    cin>>num;
    int res = getCheckPalindromeNumber(num);
    if(res == true){
        cout<<num<<" is a Palindrome number.";
    }else{
        cout<<num<<" is not a Palindrome number.";
    }
    return 0;
}
